pom.xml

```diff
@@ -76,6 +78,13 @@
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
 
+		<!-- Need to manually add as no longer part of Spring Boot -->
+		<dependency>
+		    <groupId>javax.validation</groupId>
+		    <artifactId>validation-api</artifactId>
+		    <version>2.0.1.Final</version>
+		</dependency>
+
```

Comment:
Indentation

Reply:
@rafahop  It looks fine in eclipse IDE. There is no identation issue.

Reply:
That it shows okay in your IDE does not mean it is indented correctly. Can you see it misaligned here on Github? That's because you are using tabs for indentation whereas the rest of the file is using spaces. Different IDEs can configure tabs with a different length that's why mixing tabs and spaces causes this problem. Please replace tabs with spaces 🙂

Reply:
We should be using spaces instead of tabs for indentation, so this needs to be corrected.

Reply:
I have removed the dependency and used jakarta validation as suggested in next comment.